projects/aca-content/src/lib/ui/variables/variables.scss

```diff
@@ -73,6 +75,8 @@ $defaults: (
   --theme-about-panel-border-color: $grey-background,
   --theme-about-panel-background-color: #ffffff,
   --theme-about-panel-title-color: #212121,
+  --theme-font-color-black-87-alpha: $black-87-alpha,
```

Comment:
should be domain, not embedding the color in the name

Reply:
Updated the variable name. New variable name is `--theme-datetimepicker-font-color`